source/_docs/authentication.markdown

```diff
@@ -78,11 +77,11 @@ INFO (MainThread) [homeassistant.components.http.auth] You need to use a bearer
 
 While you should hopefully be storing your passwords in a password manager, if you lose the password associated with the owner account the only way to resolve this is to delete *all* the authentication data. You do this by shutting down Home Assistant and deleting the following files from the `.storage/` folder in your [configuration folder](/docs/configuration/):
```

Comment:
@cogneato when you add the troubleshooting "I got locked out" page, please move these instructions there too and add a link from the auth page to the troubleshooting page.